anta/catalog.py

```diff
@@ -34,6 +36,21 @@
 ListAntaTestTuples = list[tuple[type[AntaTest], Optional[Union[AntaTest.Input, dict[str, Any]]]]]
 
 
+def merge_catalogs(catalogs: list[AntaCatalog]) -> AntaCatalog:
```

Comment:
let's make it a class method of AntaCatalog

Reply:
Fixed in 28e3fa985af88c21c1403185c5a07eb1cdd61a24